component/machine-sets.jsonnet

```
@@ -59,9 +59,9 @@ local machineSetSpecs = function(name, set, role)
                 antiAffinityKey: name,
               },
             } else {}
-          ) + com.makeMergeable(std.get(set, 'providerSpec', {})),
+          ),
         },
-      },
+      } + com.makeMergeable(std.get(set, 'template', {})),
     },
   };
```

Alternatively, consider reverting to the previously supported `nodeGroups.<name>.spec` instead of introducing a new parameter:

```suggestion
  } + if std.objectHas(set, 'spec') then { spec+: com.makeMergeable(set.spec) } else {};
```

Looks like this got accidentally removed in #82